backendImpl/src/test/resources/common.sh

```diff
@@ -23,6 +23,15 @@ newrepo() {
   git init $opt
 }
 
+clone() {
+  path=$1
```

Comment:
Verify that the number of params is correct, same in other similar functions while you're here